src/main.js

```diff
@@ -352,6 +352,9 @@ function handleAppGPUProcessCrashed(event, killed) {
 
 function handleAppLogin(event, webContents, request, authInfo, callback) {
   event.preventDefault();
+  if (!isTrustedURL(request.url)) {
+    return;
+  }
```

Comment:
One quick question, should the conditional be above `event.preventDefault();`? That's supposed to prevent the login request from it's default of cancelling authentication: https://electronjs.org/docs/api/app#event-login

Reply:
I don't think it matters, it just needs to happen before the event is sent back to the renderer. Unless there's something in Electron that stays open as a result of escaping that method this way.

Reply:
Ya, that's the part I'm not sure about.

Reply:
but if it is a trusted URL (like when we ever implement white list of urls, or if the server is behind a proxy), would this prevent auth as it is before checking for it?

Reply:
No, that's what `preventDefault()` is for, because the default action of Electron is to cancel authentication (see Dean's link above)